src/errorpage.cc

```diff
@@ -1149,18 +1148,10 @@ ErrorState::compileLegacyCode(Build &build)
                 p = "[no request]";
             break;
         }
-        if (request) {
-            mb.appendf(SQUIDSBUFPH " " SQUIDSBUFPH " %s/%d.%d\n",
-                       SQUIDSBUFPRINT(request->method.image()),
-                       SQUIDSBUFPRINT(request->url.originForm()),
-                       AnyP::ProtocolType_str[request->http_ver.protocol],
-                       request->http_ver.major, request->http_ver.minor);
-            request->header.packInto(&mb, true); //hide authorization data
-        } else if (request_hdrs) {
-            p = request_hdrs;
-        } else {
+        else if (request)
+            request->pack(&mb, true /* hide authorization data */);
```

Comment:
This will append CRLF that was not appended before this changes. I hope that is OK, but perhaps this PR description should mention that `%R` output has changed a little.

Reply:
Not sure documenting this is truly helpful.

Error pages contain ```<pre>%R</pre>```. Which goes from this:
```
   <pre>GET url HTTP/1.1
header
header</pre>
```
to this:
```
   <pre>GET url HTTP/1.1
header
header
</pre>
```

Reply:
added a line to description anyway.